notebooks/minigrid_colab_tutorial.ipynb

```diff
@@ -0,0 +1,558 @@
+        "</p>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Mq5SNtn4Alur"
+      },
+      "source": [
+        "Here, we jusy install RLHive. "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "iXG1wq0hHrcw",
+        "outputId": "8a6024b9-088d-4784-f2d3-b4e884ef991f"
+      },
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount(\"/content/drive\", force_remount=True).\n"
+          ]
+        }
+      ],
+      "source": [
+        "from google.colab import drive\n",
+        "drive.mount('/content/drive')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "M4tPX0_QpbOB"
+      },
+      "outputs": [],
+      "source": [
+        "## used for updating config.yaml files \n",
+        "!pip install ruamel.yaml &> /dev/null\n",
+        "\n",
+        "!pip install git+https://github.com/chandar-lab/RLHive.git@dev &> /dev/null\n",
```

Comment:
Could you add a comment as to why we install the dev branch?

Reply:
Do you think it is necessary? Since we eventually switch to the main branch!

Reply:
RIght it makes sense.